src/summary/history/conditions.resource.tsx

```diff
@@ -0,0 +1,94 @@
+import { openmrsFetch } from "@openmrs/esm-api";
+import { of } from "rxjs";
+
+export function performPatientConditionSearch(
+  patientIdentifier: string,
+  abortController: AbortController
+) {
+  return Promise.resolve(mockPatientConditionSearchResponse);
```

Comment:
Hardcoded this value since the FHIR endpoint isn't ready and the history of the condition emrapi endpoint has an error